### PR TITLE
Add kde code for calculating the DM of the MW halo

### PR DESCRIPTION
**Pull request by EmPlatts**

### CODE REVIEW

File: frb/dm_kde/transient_data/frbcat.csv [hunk 1]
```diff
@@ -0,0 +1,111 @@
+﻿"frb_name","utc","telescope","rop_raj","rop_decj","rop_gl","rop_gb","rmp_dm","rmp_width","rmp_snr"
```
Comment (profxj) [inline frb/dm_kde/transient_data/frbcat.csv:1]: Can you move this file into frb/data/FRBs ?

And rename it like the other ones in there.
Thanks!

File: frb/dm_kde/transient_data/psrcat.csv [hunk 2]
```diff
@@ -0,0 +1,2705 @@
+#,NAME,dm,P0,RAJD,DECJD
```
Comment (profxj) [inline frb/dm_kde/transient_data/psrcat.csv:1]: Is all this data in https://github.com/FRBs/pulsars
already?  I suspect so.  If so, remove this file and
just link your code to that other Repo.

File: frb/dm_kde/transient_kdes.py [hunk 3]
```diff
@@ -0,0 +1,133 @@
+""" Definitions relating to the observed DM distributions of pulsars and FRBs.
+Standard KDE is used for pulsars and varying assymetric KDE for FRBs.
+To use this code, please download the asymmetric_kde package from
+https://github.com/tillahoffmann/asymmetric_kde [Hoffman and Jones, 2015] """
+
+from asymmetric_kde import ProperGammaEstimator
+import numpy as np
+import pandas as pd
+from scipy.signal import argrelextrema
+from pdf_fns import make_kde_funtion
```
Comment (profxj) [inline frb/dm_kde/transient_kdes.py:10]: is this in a standard anaconda install?  if not, 
we need to add it to the optional install notes, i.e.
https://github.com/FRBs/FRB/blob/master/docs/installing.rst

Comment (profxj) [reply frb/dm_kde/transient_kdes.py:10]: Same with `asymmetric_kde`

**Review by profxj — CHANGES_REQUESTED** [review]
Thanks so much for this @EmPlatts 

I have a few requests of you.  Let me know if any of them are onerous.
Thanks!

File: frb/dm_kde/transient_data/frbcat.csv (hunk 1 above)
Comment (EmPlatts) [reply frb/dm_kde/transient_data/frbcat.csv:1]: Done :)

File: frb/dm_kde/transient_data/psrcat.csv (hunk 2 above)
Comment (EmPlatts) [reply frb/dm_kde/transient_data/psrcat.csv:1]: I've added a pull request to add the file to the repo (v1.61)

File: frb/dm_kde/transient_kdes.py (hunk 3 above)
Comment (EmPlatts) [reply frb/dm_kde/transient_kdes.py:10]: pdf_fns is just a file in the repo

Comment (EmPlatts) [reply frb/dm_kde/transient_kdes.py:10]: Will add to docs for asymmetric kde, thanks!

Comment (EmPlatts) [reply frb/dm_kde/transient_kdes.py]: Done :)

**Comment by EmPlatts** [comment]
@profxj Have made the changes, thank you!

**Pull request merged**